dict/prh_bengo4.yml

```
@@ -7,22 +7,25 @@ rules:
     patterns: 弁コム
     prh: サービス名の場合は略さず表記します
   - expected: 弁護士ドットコムLIBRARY
-    patterns: /弁護士ドットコム(Library|library)/
+    patterns: /弁護士ドットコム(\s?[Ll]ibrary)/
     prh: サービス名（英語表記）
   - expected: 弁護士ドットコムライブラリー
     patterns: /弁護士ドットコムライブラリ([^ー])/
     prh: カタカナ表記時は長音（ー）あり
-  - expected: 弁護士ドットコムライブラリー
-    patterns: /弁護士ドットコムライブラリ([^ー])/
-    prh: カタカナ表記時は長音（ー）あり
+  - expected: 業務支援システム
+    patterns: /業シス/
```

```suggestion
    patterns: 業シス
```

正規表現をつかっていないので囲む必要はないかと思いました。